en/firefox/enterprise.ftl

```diff
@@ -43,12 +43,12 @@ firefox-enterprise-release-notes = Release Notes
 firefox-enterprise-documentation-and-community = Documentation and Community Support
 
 # Variables:
-#   $url (url) - link to https://github.com/mozilla/policy-templates/blob/master/mac/org.mozilla.firefox.plist
+#   $promise (url) - link to https://github.com/mozilla/policy-templates/blob/master/mac/org.mozilla.firefox.plist
 firefox-enterprise-sample-plist-for-configuration = Sample <a href="{ $url }">plist for configuration profile</a>
 
 firefox-enterprise-pkg-installer = PKG installer
 firefox-enterprise-windows-32-bit = { -brand-name-windows } 32-bit
 
 # Variables:
-#   $firefox_all (url) - link to https://www.mozilla.org/firefox/all/#product-desktop-esr
+#   $promise (url) - link to https://www.mozilla.org/firefox/all/#product-desktop-esr
```

Comment:
@stevejalim is this change here intentional? The comment here is changed to `$promise` but the actual variable in the string is `$firefox_all`?

Reply:
Oh, same for the other variables above?

Reply:
@peiying2 @stevejalim looks like this is reverting a fix because the source `.ftl` files in bedrock weren't updated. I'm going to close this PR and open a new fix in bedrock.